config/config.yaml

```diff
@@ -0,0 +1,90 @@
+# netbox device attributes that should be used as filters for devices query.
+# these translate directly into the return fields from the netbox json API.
+# If no filter is specified, by default only devices with Active (1) status are pulled.
+filters:
```

Comment:
the name should be descriptive. As far as I understand, this filter is applied to device attributes in netbox. I suggest name "netbox_device_attribute_filter"